### PR TITLE
delete does not require for the shard to be load

### PR DESCRIPTION
**Pull request by hermeGarcia**
### Description
Deleting a shard required for it to be loaded. This causes problems if the shard is corrupted, this PR fixes the problem.

### How was this PR tested?
Local tests

### CODE REVIEW

File: nucliadb_node/src/writer/grpc_driver.rs [hunk 1]
```diff
@@ -101,23 +101,19 @@ impl NodeWriter for NodeWriterGRPCDriver {
         self.instrument(&request);
 
         info!("gRPC delete_shard {:?}", request);
-
+        // Deletion does not requiere for the shard
```
Comment (alekece) [inline nucliadb_node/src/writer/grpc_driver.rs:104]: Little typo here `requiere` => `require`

File: nucliadb_node/src/writer/grpc_driver.rs [hunk 2]
```diff
@@ -130,8 +126,9 @@ impl NodeWriter for NodeWriterGRPCDriver {
 
         info!("gRPC delete_shard {:?}", request);
 
+        // Deletion and upgrade do not requiere for the shard
```
Comment (alekece) [inline nucliadb_node/src/writer/grpc_driver.rs:129]: Ditto

**Review by alekece — APPROVED** [review]
Nice catch !

File: nucliadb_node/src/writer/grpc_driver.rs (hunk 1 above)
Comment (hermeGarcia) [reply nucliadb_node/src/writer/grpc_driver.rs]: Ouch true! Thanks for noticing, fixed it at [ad5a4c3](https://github.com/nuclia/nucliadb/pull/462/commits/ad5a4c34b67331b5357922d335e22f3c09505558)

**Pull request merged**